docs/rfcs/0022-beneficiary-merging.md

```diff
@@ -196,14 +210,154 @@ Patient 5 is replaced by patient 6
 }
 ```
 
-#### Option 2 - Automatic Replacement
+#### Option 4 - Links (See Also)
 
 Using the following data:
 
-| bene_id | MBI         | xref_id | xref_sw |
-| ------- | ----------- | ------- | ------- |
-| 5       | 1S00EU8FF08 | 2       | Y       |
-| 6       | 1S00EU8FF09 | 2       | N       |
+| bene_id | MBI         | xref_id | xref_sw | kill_credit |
+| ------- | ----------- | ------- | ------- | ----------- |
+| 5       | 1S00EU8FF08 | 2       | Y       | null        |
+| 6       | 1S00EU8FF09 | 2       | N       | null        |
+
+Searching by id (bene_id) will return only the matched record.
+Searching by identifier (MBI) will return both resources.
+In both cases, the response will contain a `link` field marked as `seealso`, which links to the other resorce.
+
+As opposed to the `replaced-by`/`replaces` solution, this does not attempt to convey identity, which is helpful in the event that an un-merge occurs.
+
+We will still need an alternative way to convey which resource is the most current one.
+This may need to be accomplished with an extension.
+
+Searching for id=5 yields:
+
+```json
+{
+  "resourceType": "Bundle",
+  "id": "720151cf-dfc9-4516-ab39-2925100c7429",
+  "meta": {
+    "lastUpdated": "2024-08-09T12:44:27.371-07:00"
+  },
+  "type": "searchset",
+  "total": 1,
+  "link": [
+    {
+      "relation": "self",
+      "url": "https://prod.bfd.cms.gov/v2/fhir/Patient?_format=json&_id=5"
+    }
+  ],
+  "entry": [
+    {
+      "resource": {
+        "resourceType": "Patient",
+        "id": "5",
+        "link": [
+          {
+            "other": {
+              "reference": "Patient/6"
+            },
+            "type": "seealso"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+Searching for id=6 yields:
+
+```json
+{
+  "resourceType": "Bundle",
+  "id": "720151cf-dfc9-4516-ab39-2925100c7429",
+  "meta": {
+    "lastUpdated": "2024-08-09T12:44:27.371-07:00"
+  },
+  "type": "searchset",
+  "total": 1,
+  "link": [
+    {
+      "relation": "self",
+      "url": "https://prod.bfd.cms.gov/v2/fhir/Patient?_format=json&_id=6"
+    }
+  ],
+  "entry": [
+    {
+      "resource": {
+        "resourceType": "Patient",
+        "id": "6",
+        "link": [
+          {
+            "other": {
+              "reference": "Patient/5"
+            },
+            "type": "seealso"
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+Searching for either MBI 1S00EU8FF08 or 1S00EU8FF09 will yield both records:
+
+```json
+{
+  "resourceType": "Bundle",
+  "id": "720151cf-dfc9-4516-ab39-2925100c7429",
+  "meta": {
+    "lastUpdated": "2024-08-09T12:44:27.371-07:00"
+  },
+  "type": "searchset",
+  "total": 1,
```

Comment:
Small catch but total is 2 here

Reply:
Can you refresh? The line number here doesn't match up with the current version.

Reply:
This is only a single resource so I'm assuming this comment is outdated.